docs/architecture/gas/estimator.md

```diff
@@ -20,100 +20,9 @@ clients.
 The estimator code is part of the nearcore repository in the directory
 [runtime/runtime-params-estimator](https://github.com/near/nearcore/tree/master/runtime/runtime-params-estimator).
```

Comment:
not part of the PR but should be update the link here to not point to master like we do for other links?

Reply:
Sounds good.